### PR TITLE
7136 applicant info page 2

### PR DESCRIPTION
**Pull request by rianfowler**
2018-01-25-12-31-localhost_3001

### CODE REVIEW

**Comment by rianfowler** [comment]
LMK if there are other types of tests I need to write for this and who else I should add as a reviewer.

File: src/js/vre/chapter36/config/form.js [hunk 1]
```diff
@@ -88,30 +99,72 @@ const formConfig = {
               }
             }
           }
+        },
+        nonVeteranInformation: {
+          title: 'Applicant information',
+          path: 'non-veteran-information',
```
Comment (rianfowler) [inline src/js/vre/chapter36/config/form.js]: LMK if I need to update this path. I've seen other patterns follow a `chapter/page` format.

Comment (jbalboni) [reply src/js/vre/chapter36/config/form.js]: @stacyhlinn Do you have a good name for this page? We can use that come up with the url.

File: src/js/vre/chapter36/config/form.js [hunk 2]
```diff
@@ -88,30 +99,72 @@ const formConfig = {
...
+            receivedPamphlet: {
+              'ui:title': 'Have you received a pamphlet explaining survivors\' and dependents\' educational assistance benefits?',
+              'ui:widget': 'yesNo'
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              applicantSocialSecurityNumber,
+              applicantGender: {
```
Comment (jbalboni) [inline src/js/vre/chapter36/config/form.js]: We try to use the properties from the schema as much as possible. In this case, you could import `applicantGender` from the schema properties or `gender` from the schema definitions and use it here. If you do the former, you'll have to put the `gender` definition in `defaultDefinitions`. 

Then, to set the labels, you can put them in the uiSchema for this field:

```js
applicantGender: {
  'ui:title': 'Gender',
  'ui:widget': 'radio',
  'ui:options': {
    labels: genders
  }
}
```

Our select and radio widgets support getting labels both from here and in the `enumNames` property of the schema.

File: src/js/vre/chapter36/config/form.js [hunk 3]
```diff
@@ -41,6 +50,8 @@ const formConfig = {
         applicantInformation: {
           title: 'Applicant information',
           path: 'applicant-information',
+          initialData: {},
```
Comment (jbalboni) [inline src/js/vre/chapter36/config/form.js]: You could remove this.

**Review by jbalboni — APPROVED** [review]
LGTM, just had a couple small comments.

File: src/js/vre/chapter36/config/form.js [hunk 4]
```diff
@@ -88,30 +99,72 @@ const formConfig = {
...
+            seekingRestorativeTraining: {
+              'ui:title': 'Are you a child who is at least 14 years old, a spouse, or a surviving spouse with a disability and looking for special restorative training?',
+              'ui:widget': 'yesNo'
+            },
+            seekingVocationalTraining: {
+              'ui:title': 'Are you a child, a spouse, or a surviving spouse with a disability and looking for special vocational training',
+              'ui:widget': 'yesNo'
+            },
+            receivedPamphlet: {
+              'ui:title': 'Have you received a pamphlet explaining survivors\' and dependents\' educational assistance benefits?',
```
Comment (cehsu) [inline src/js/vre/chapter36/config/form.js]: We tend to use curly apostrophes (’) in general.

File: src/js/vre/chapter36/config/form.js (hunk 1 above)
Comment (cehsu) [reply src/js/vre/chapter36/config/form.js]: If only dependents can claim this (besides the Veteran), could we call this dependent information?

**Review by cehsu — COMMENTED** [review]
LGTM. Pending the decision regarding the path, and the recommended changes to the gender radio widget.

**Review by cehsu — APPROVED** [review]

File: src/js/vre/chapter36/config/form.js (hunk 1 above)
Comment (rianfowler) [reply src/js/vre/chapter36/config/form.js]: @stacyhlinn is out but I like the idea to update it to dependent-information and will do that. We can update it later if we need to.

**Pull request merged**